### DISCUSSION TITLE
precipitation_sum vs precipitation_probability_max

### DISCUSSION THREAD

**Discussion by Gilbert54** [open-meteo/open-meteo#786] [Q&A]
Hi,
I'm n ot a weather frog ;-), but I intend to use meteo data to get a general impression how the weather will be the remainder of today and the next few days. What I'm particularly interested in is the probablity for a rainy day.
I currently get precipitation_sum = 0.00 and precipitation_probability_max = 39 for a particular day. How do I have to read this? 
There is a 39% probability that it's going to rain with an amount of 0 mm? Probably not...
Or is it a 39% probabilty for rain and an unknown quantity? maybe...
In my mind 39% is a fair chance for a wet day? Surely that % must be based on an amount of rain which is likely to fall?
Please help me understand.

**Comment by patrick-zippenfenig** [comment]
Hi, see here: https://github.com/open-meteo/open-meteo/issues/783

**Comment by Leftium** [comment]
I'm not sure if Open-Meteo is the same, but when using other API's that provide both precipitation amount and probability (DarkSky and OpenWeather), I have found probability and amount are fairly correlated, but amount provides more useful information (in terms of deciding if I am OK without an umbrella). (See my comment here: https://github.com/open-meteo/open-meteo/issues/783#issuecomment-2107355426)

In general, I believe the precipitation probability is the chance that *any* amount of rain will fall *anywhere* within a certain area during that time period. Which is not quite the same as the chance that rain will fall at *your* location (so that you, yourself, will experience rain).

Generally, I think if the chance of precipitation is higher, so is the amount. But that may not always be the case.

https://briefsky.app/ nicely graphs both the precipitation probability and amount, based on Open-Meteo data. Compare with the actual weather you experience to get a feel for how to read these values.

![image](https://github.com/open-meteo/open-meteo/assets/381217/45656572-e190-48b4-8901-752f081d5684)

**Comment by Gilbert54** [comment]
John-Kim,
thanks for taking the time to respond. Glad to see that I'm not the only
one who is confused by what intuitively appears to be a contradiction. I
intended to take the amount of rain (or any other form of precipitation) as
a guide to decide if I need an umbrella or not.
My expection is that I get general weather information (at least info about
sunny skies or rain) for the area close to the coordinates (lon, lat) which
I provide in the API call. Otherwise the whole excercise would be pointless?

Op ma 13 mei 2024 om 13:55 schreef John-Kim Murphy ***@***.***
